src/app/containers/reference-form-dialog/reference-form-renderer.tsx

```diff
@@ -66,6 +67,32 @@ export const DateInput: React.FC<ReferenceInputTypeProps<string>> = (props) => {
   );
 };
 
+export const SpecificUseInput: React.FC<ReferenceInputTypeProps<string>> = (props) => {
```

Comment:
Label is rendering incorrectly for me

![image](https://user-images.githubusercontent.com/52923935/89511643-1b52d380-d7ca-11ea-8a93-3d80bdb89602.png)

Reply:
Also, missing placeholder 'Please Select' text although that appears once I open the options list.